components/navbar/me.tsx

```diff
@@ -9,12 +9,13 @@ import {
   Divider,
 } from '@nextui-org/react';
 import { useState, useEffect, useContext } from 'react';
-import { GoPersonFill } from 'react-icons/go';
+import { GoPersonFill, GoGear } from 'react-icons/go';
 import Logout from '@/components/navbar/me/logout';
 import { AuthContext } from '@/contexts/auth';
 import { loginThroughTool } from '@/actions/auth/auth';
 import { getMe } from '@/actions/me/me';
 import Loading from '@/components/loading';
+import Link from 'next/link';
```

Comment:
Nit: Can we keep consistency and use `NextLink` throughout the app?

Reply:
done!